dev-requirements.txt

```diff
@@ -1,5 +1,13 @@
-pipenv==2022.4.8; python_version >= '3.6'
+pipenv==2022.4.8; python_version == '3.8'
```

Comment:
why 3.6 and 3.7 are not included? (I might be missing some context here)

Reply:
They were not previously included in the pipeline tests either. Both the circleci config and this file are only related to these tests. I suppose since these are pretty old, deprecated versions of python it's not really necessary to also use them for these tests.

Reply:
> They were not previously included in the pipeline tests either. Both the circleci config and this file are only related to these tests. I suppose since these are pretty old, deprecated versions of python it's not really necessary to also use them for these tests.

Thank you for the context, if we are deprecating them, then I'd suggest to update our "README" in terms of supported versions of python